tests/routes/artifact_test.py

```diff
@@ -21,13 +21,6 @@ def test_artifact_get_none(self):
             .expect(404, self.RESPONSE_404) \
             .get(headers=self.auth)
 
-    def test_no_bucket(self):
-        self.route_tester \
-            .artifact() \
-            .route_params(bucket_name="ufo", path="john-travolta") \
```

Comment:
I feel like you could have at least kept my reference to John Travolta... I mean seriously have some reverence!